README.md

```diff
@@ -54,7 +57,27 @@ For Unix systems extract the archive to a directory on in your path, for example
 $ tar -xf /your/download/location/dolt-darwin-amd64.tar.gz -C /usr/local/bin
 $ ln -s /usr/local/lib/dolt /usr/local/lib/dolt/bin/dolt
 ```
-Verify that your installation has succeeded as follows:
+
+### From Source
+Alternatively clone this repository and then, assume you cloned the repository into your home directory:
+```
+$ cd ~/dolt/go
+$ go install ./cmd/dolt
+[...]
+$ go install ./cmd/git-dolt
+[...]
+$ go install ./cmd/git-dolt-smudge
+[...]
+```
+This will install the the requisite binaries at `$GOROOT/bin`, which defaults to `$HOME/go`, thus you should see something like (unless you set `$GOROOT` to something else):
```

Comment:
```suggestion
This will install the requisite binaries at `$GOROOT/bin`, which defaults to `$HOME/go`, thus you should see something like (unless you set `$GOROOT` to something else):
```